cartridges/int_affirm/cartridge/scripts/init/initAffirmServices.js

```diff
@@ -26,6 +26,12 @@ var commonCreateRequest = function (svc, args) {
     svc.addHeader('Content-Type', 'application/json');
     svc.addHeader('Authorization', getAuthHeader(require('dw/system/Site').current));
     svc.addHeader('Idempotency-Key', getIdempotencyKey());
+
+    var affirmData = require('*/cartridge/scripts/data/affirmData');
+    var country_code = affirmData.getCountryCode();
+    if (country_code && country_code !== undefined) {
```

Comment:
does this check suffice all use cases, does it need typeof country_code !== 'undefined' also?

Reply:
updated